api/performance_test.go

```diff
@@ -6,7 +6,7 @@ import (
 	"github.com/ghodss/yaml"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
-	"github.com/openshift-kni/performance-addon-operators/pkg/apis/performance/v1"
+	"github.com/openshift-kni/performance-addon-operators/api/v1"
```

Comment:
this is a breaking change considering we want to go GA with 4.6

Reply:
this is a 4.7 PR, I understand we already have a 4.6 branch. @cynepco3hahue am I right?

Reply:
you are right, but nevertheless breaking change should be carefully evaluated after we do a stable release. People expect stability from stable products, and breaking changes... well, go in the opposite direction :)

Reply:
While I fully agree, the PAO is no yet widely used and the change will not break PAO on upgrade

Reply:
Correct, but it will break any consumer of its api package. Hence, we need to evaluate this carefully and most likely a t least advertise it in the future release notes.

Reply:
Agreed, release nodes is a must

Reply:
Added to https://docs.google.com/document/d/1vjLAgnWyMI5La-aoeXUtFGdHN6_em9TjTYFOL_1VlrM